File: Shaima Haque/Basics of Programming/is_prime.cpp
#include <iostream>
using namespace std;
int main(int argc, char **argv){
    int t;
    cin >> t;
    int n;
    //write your code here
    for(int i = 0;i<t;i++){
        cin >> n;
        int flag  = 0;
        if(n <= 1){
            flag = 1;
        }else{
        for(int j = 2;j<=(int)n/2;j++){
            if(n%j == 0){
                flag = 1;
                break;
            }
        }
        }
        if(flag == 1){
            cout << "not prime" <<endl;
        }else{
            cout << "prime" << endl;
        }
    }
}

File: V.Vennela/Basics of Programming/IsPrime.java
import java.util.*;
  
  public class Main{
  
  public static void main(String[] args) {
      Scanner scn = new Scanner(System.in);
  
       // write ur code here
       int t = scn.nextInt();
       while(t!=0)
       {
           int n = scn.nextInt();
           boolean flag=true;
          
           for(int i=2;i<n;i++)
           {
               if(n%i==0)
                flag =false;
           }
            if(n<0)
             {
                 flag = false;
             }
           if(flag==true)
            System.out.println("prime");
           else
            System.out.println("not prime");
           t--;
       }
  
   }
  }
